substrate/client/network/src/litep2p/mod.rs

```diff
@@ -790,6 +801,45 @@ impl<B: BlockT + 'static, H: ExHashT> NetworkBackend<B, H> for Litep2pNetworkBac
 							self.peerstore_handle.add_known_peer(peer.into());
 						}
 					}
+					Some(DiscoveryEvent::FindNodeSuccess { query_id, target, peers }) => {
+						match self.pending_queries.remove(&query_id) {
```

Comment:
Would it make sense to report these peers to the peerstore handle as well? Similar to `DiscoveryEvent::RoutingTableUpdate { peers }`, since this might boost up the random walks network discoverability?

Reply:
In theory yes, but the impact should be quite limited as this is only used by DHT bootnodes, and it's hard to imagine the consequences of adding not random nodes, but the ones discovered during the bootnode lookup (like, if the connectivity going to be biased toward peers closer to bootnode providers?), so I'd rather keep the random walk separate.